update_module_details.php

```diff
@@ -0,0 +1,305 @@
+<?php
+echo "\n\nUpdating pipeline details - " . date("Y-m-d h:i:s") . "\n";
+
+// Allow PHP fopen to work with remote links
+ini_set("allow_url_fopen", 1);
+// Load yaml parser
+require "vendor/autoload.php";
+use Symfony\Component\Yaml\Yaml;
+
+// Get auth secrets
+$config = parse_ini_file("config.ini");
+$gh_auth = base64_encode($config['github_username'] . ':' . $config['github_access_token']);
+$conn = mysqli_connect($config['host'], $config['username'], $config['password'], $config['dbname'], $config['port']);
+
+if ($conn === false) {
+    die("ERROR: Could not connect. " . mysqli_connect_error());
+}
+
+//
+//
+// GitHub API calls
+//
+//
+
+function github_query($gh_query_url)
```

Comment:
Do we not have already have a helper function to do this stuff somewhere? Might only be in nf-core/tools that I'm thinking of.

Could be good to centralise it though, as I guess we have this duplicated in a bunch of places.. A `utils.php` file or something (if not `functions.php`)

Reply:
yep, agree with functions.php. We have a similar function in tools, but that's in python...